### PR TITLE
Allow expand "AccordionItem" programmatically

### PR DESCRIPTION
**Pull request by epotockiy**
Add `activeItems` prop realization for programmatically controlling of expanded Accordion items #12

### CODE REVIEW

**Comment by thibaudcolas** [comment]
👍 related: #12

@vincentaudebert do you have time to review this in the near future?

**Comment by vincentaudebert** [comment]
I will try to fit this in this week or the next one. Exciting!

**Comment by vincentaudebert** [comment]
@epotockiy would it be possible that you provide some examples or ideally that you test your changes so we get 100% coverage?
Thanks.

**Comment by epotockiy** [comment]
@vincentaudebert I'll try to do it.

**Comment by vincentaudebert** [comment]
Any progress @epotockiy ?

**Comment by epotockiy** [comment]
@vincentaudebert I added some tests for new functionality, could you review this?

**Comment by vincentaudebert** [comment]
It's looking really good to me @epotockiy ! Thanks a lot for your amazing work!
@thibaudcolas could you give a final review? Thanks.

File: README.md [hunk 1]
```diff
@@ -93,6 +93,12 @@ ReactDOM.render(<Example />, document.querySelector('[data-mount]'));
           <td>accordion</td>
           <td>CSS class(es) applied to the component</td>
       </tr>
+      <tr>
+          <td>activeItems</td>
+          <td>Array</td>
+          <td>[]</td>
+          <td>Indexes (or custom keys) to pre expand items. Can be changed dynamically. Doesn't have the priority against `AccordionItem - expanded` on first render.</td>
```
Comment (thibaudcolas) [inline README.md]: Would be nice to have some other documentation mentioning this `customKey` prop.

File: src/Accordion/accordion.js [hunk 2]
```diff
@@ -28,17 +34,38 @@ class Accordion extends Component {
         this.renderItems = this.renderItems.bind(this);
     }
 
+    componentWillReceiveProps(nextProps) {
+        if (!isArraysEqual(nextProps.activeItems, this.state.activeItems)) {
```
Comment (thibaudcolas) [inline src/Accordion/accordion.js]: Rename this to `isArraysEqualShallow` to make sure there is no confusion about what is going on here?

File: src/utils/index.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+// eslint-disable-next-line
```
Comment (thibaudcolas) [inline src/utils/index.js]: This override doesn't seem necessary – I imagine it's to circumvent the rule "no solo `const` export in a file". Could this be moved as an `export default` in a separate file instead?

**Comment by vincentaudebert** [comment]
Merged. Thanks again @epotockiy will be released into `0.6.0`

**Pull request merged**